src/web/components/SmallHeadline.tsx

```diff
@@ -95,9 +107,23 @@ export const SmallHeadline = ({
     showQuotes = false,
     size = 'xxsmall',
     coloured = false,
-    linkTo,
+    link,
 }: Props) => {
-    const Headline = linkTo ? 'a' : 'span';
+    // Compose object with attributes that are only relevant when rendering a link tag
```

Comment:
Decided to treat the existence of `to inside `link` as a pre-condition for the other `link` properties to take any effect.